### PR TITLE
Add support for acl dryrun command

### PR DESCRIPTION
**Pull request by aksh-02**
Fixes https://github.com/redis/go-redis/issues/2381

### CODE REVIEW

**Comment by aksh-02** [comment]
@monkey92t @chayim please review these changes

File: commands.go [hunk 1]
```diff
@@ -450,6 +450,8 @@ type Cmdable interface {
 	GeoSearchStore(ctx context.Context, key, store string, q *GeoSearchStoreQuery) *IntCmd
 	GeoDist(ctx context.Context, key string, member1, member2, unit string) *FloatCmd
 	GeoHash(ctx context.Context, key string, members ...string) *StringSliceCmd
+
+	AclDryRun(ctx context.Context, username, command string, args ...interface{}) *StringCmd
```
Comment (monkey92t) [inline commands.go:454]: // or args or commands
should: ACLDryRun(ctx, username, command ...interface)

for us, there is no need to distinguish between command and args.  It's ACL, not Acl, just like CNCF and not Cncf.

**Review by monkey92t — APPROVED** [review]
tks..

File: commands.go (hunk 1 above)
Comment (aksh-02) [reply commands.go]: comment incorporated

**Pull request merged**